rollup.config.js

```diff
@@ -1,12 +1,23 @@
-import uglify from "rollup-plugin-uglify"
+import babel from 'rollup-plugin-babel'
+import uglify from 'rollup-plugin-uglify'
 
 export default {
```

Comment:
@lukejacksonn Can you completely remove this file? And use the same approach on Hyperapp. See Hyperapp's  package.json.